### ISSUE TITLE
PSR12.Files.OpenTag incorrectly identifies PHP file with only an opening tag

### ISSUE THREAD

**Issue by grantlucas** [squizlabs/PHP_CodeSniffer#2664]
When trying out the latest PSR-12 support, I found that `PSR12.Files.OpenTag.NotAlone` was triggering on PHP files where there is _just_ an opening PHP tag. We have some of these in our project due to framework reasons. It's not great but kind of required.

## Triggers error

```php
<?php
```

> [phpcs] Opening PHP tag must be on a line by itself (PSR12.Files.OpenTag.NotAlone)

## Doesn't trigger error

```php
<?php

// comment
```

Please correct me if I'm wrong but I don't see anywhere in PSR-12 where a PHP file can't just have an opening tag, however useless.

**Comment by gsherwood** [comment]
Thanks for reporting this. A fix will be in 3.5.2.

**Comment by grantlucas** [comment]
😮 🎉  Fasted bug fix ever. Thanks!

**Comment by gsherwood** [comment]
⏱️ You're welcome
